### PR TITLE
FIX jepst Dockerfile bashrc to right place

### PR DESCRIPTION
**Pull request by jepst**
Commands echoed to student bashrc were going to the copy in /etc, not the student's home directory

### CODE REVIEW

**Comment by wabscale** [comment]
Since the home directory is going to have the volume mounted on top of it, that bashrc file will essentially disappear when the ide starts. 

Our solution to this is to have an [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) that runs before the ide starts to setup the volume. There are a couple of things that this script does, like cloning the student github repo if applicable. In this step, we [copy over files from /etc/skel if they do not already exist in the volume](https://github.com/GusSand/Anubis/blob/master/theia/init/entrypoint.sh#L15). 

So basically the first time that a student opens their ide, the bashrc and profile from /etc/skel will be copied into their persistent volume. Because of this, I think that the existing setup does what you want it to. I will push a commit that adds that python alias to the init image so that it will be available to your students.
